core-bundle/src/Doctrine/Schema/DcaSchemaProvider.php

```diff
@@ -88,6 +88,9 @@ public function appendToSchema(Schema $schema): void
                         $options['platformOptions']['collation'] = $options['customSchemaOptions']['collation'];
                     }
 
+                    // Deprecated in doctrine/dbal 3.x and removed in 4.x
+                    unset($options['customSchemaOptions']);
```

Comment:
should we add a deprecation or something like that if it is still used in the DCA?

Reply:
I think so, but I am not sure if simply unsetting it is the correct solution anyway. We automatically map

* `$options['customSchemaOptions']['charset']` » `$options['platformOptions']['charset']`
* `$options['customSchemaOptions']['collation']` » `$options['platformOptions']['collation']`

But there is also support for 

* `$options['customSchemaOptions']['unique']`
* `$options['customSchemaOptions']['case_sensitive']`

in our `DcaSchemaProvider` and `Widget` class.

Reply:
I think we should rewire them all from `customSchemaOptions` to `platformOptions` and deprecate `customSchemaOptions`.

Reply:
👍 if `platformOptions` support these properties